aws/resource_aws_wafregional_xss_match_set.go

```diff
@@ -0,0 +1,173 @@
+package aws
+
+import (
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/waf"
+	"github.com/hashicorp/errwrap"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsWafRegionalXssMatchSet() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsWafRegionalXssMatchSetCreate,
+		Read:   resourceAwsWafRegionalXssMatchSetRead,
+		Update: resourceAwsWafRegionalXssMatchSetUpdate,
+		Delete: resourceAwsWafRegionalXssMatchSetDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"xss_match_tuples": &schema.Schema{
```

Comment:
As we [discussed](https://github.com/hashicorp/terraform/pull/13705#discussion_r111887491) in the other resource do you mind renaming this field to singular? i.e. `xss_match_tuple`?